### DISCUSSION TITLE
I don't know reasons of this errors: system_processing_system7_0_0_synth_1

### DISCUSSION THREAD

**Discussion by congchungmta92** [XyleMora/EBAZ4205#8] [Embedded Software Design Forum]
[Board 49-71] The board_part definition was not found for miner.ebang.com.cn:ebaz4205:part0:1.0. The project's board_part property was not set, but the project's part property was set to xc7z010clg400-1. Valid board_part values can be retrieved with the 'get_board_parts' Tcl command. Check if board.repoPaths parameter is set and the board_part is installed from the tcl app store.

Please help me. I use Vivado 2022.1

**Comment by XyleMora** [comment]
Sorry for a late reply. I tried board files with Vivado 2021.2 and it works fine. I will try it later with version 2022.1. Meanwhile please look at attached link, maybe this will help: https://support.xilinx.com/s/question/0D52E00006lLBqpSAG/setting-board-file-fails-with-error-board-4971-the-boardpart-definition-was-not-found?language=en_US
